cocotb/share/lib/gpi_log/gpi_logging.c

```diff
@@ -207,7 +212,7 @@ void gpi_log(const char *name, long level, const char *pathname, const char *fun
     }
 
     // Log function args are level, filename, lineno, msg, function
```

Comment:
Please adjust this comment for the added name argument.

Reply:
Will do